manifests/deis-monitor-prometheus-rc.yaml

```diff
@@ -7,11 +7,11 @@ metadata:
 spec:
   replicas: 1
   selector:
-    app: deis-monitor-prometheus
```

Comment:
internally for helm we've been using `app` as the label instead of `name` so it's not confused with the [`name` metadata](http://kubernetes.io/v1.0/docs/user-guide/walkthrough/README.html#pod-definition). Is there any reason this is changing?